invenio_circulation/config.py

```diff
@@ -102,6 +106,11 @@
         duration_validate=is_loan_duration_valid,
         item_available=is_item_available
     ),
+    extension=dict(
+        from_end_date=extend_from_end_date,
+        duration_default=get_default_extension_duration,
```

Comment:
I would rename `duration_default` by `duration`. We can do the same for `checkout` policy.